### PR TITLE
run rustfmt on test/run-fail folder

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

File: src/test/run-fail/meta-revision-bad.rs [hunk 1]
```diff
@@ -13,10 +13,18 @@
 
 // revisions: foo bar
 // should-fail
-//[foo] error-pattern:bar
-//[bar] error-pattern:foo
+// [foo] error-pattern:bar
+// [bar] error-pattern:foo
```
Comment (alexcrichton) [inline src/test/run-fail/meta-revision-bad.rs]: I think this may break these tests because compiletest currently searches for literally "//["

**Comment by alexcrichton** [comment]
cc @nrc

File: src/test/run-fail/meta-revision-bad.rs (hunk 1 above)
Comment (nrc) [inline src/test/run-fail/meta-revision-bad.rs]: Probably best to manually fix this for now. I intend to change the default to not touch comments in the next release of rustfmt, so long term (well, medium term) it shouldn't be a problem

**Comment by srinivasreddy** [comment]
@alexcrichton  @nrc  manually fixed. Take a look.

**Comment by alexcrichton** [comment]
er, that was just muscle memory...

r? @nrc

**Pull request merged**